aten/src/ATen/native/quantized/cudnn/utils.h

```diff
@@ -168,6 +170,7 @@ struct TORCH_API PackedConvWeightCudnn : public ConvPackedParamsBase<kSpatialDim
   int64_t groups_;
   bool transpose_;
   c10::QScheme q_scheme_;
+  int64_t output_channels_;
```

Comment:
can we make this more explicit? like unpaded_output_channels_ or original_output_channels_?